apis/externalsecrets/v1beta1/secretstore_types.go

```diff
@@ -50,7 +50,12 @@ type ClusterSecretStoreCondition struct {
 	NamespaceSelector *metav1.LabelSelector `json:"namespaceSelector,omitempty"`
 
 	// Choose namespaces by name
+	// +optional
 	Namespaces []string `json:"namespaces,omitempty"`
+
+	// Choose namespaces by using regex matching
+	// +optional
+	NamespacesRegexes []string `json:"namespacesRegexes,omitempty"`
```

Comment:
Naming: shouldn't it be `namespaceRegexes` (singular + plural)? Sounds better to me at least :D Not sure though, not an englishman :laughing:

Reply:
Yeah you're right. I'll rename it later today. 🤣

Reply:
Done. :0